GenerateAllSolution.ps1

```diff
@@ -46,7 +46,7 @@ Param (
     [string[]]$Components = @('all'),
 
     [Alias("winui")]
-    [int]$WinUIMajorVersion = 2,
```

Comment:
Weird, I ran the `GenerateAllSolution.bat` file in the root, and it still was in 2 mode?

Reply:
Odd, I'm seeing that too. Not seeing the same behavior with the single-component solution generation.

Reply:
Fixed and pushed, looks like one of the cherry-picked commits from this PR wasn't included in the update/tooling branch we use for upstream testing.